mail/getmail6/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        getmail6 getmail6 6.19.03 v
+github.tarball_from tarball
```

Comment:
`tarball` is the default value of `github.tarball_from` and is deprecated and should not be used. Use `archive` if no release tarballs are available. The checksums will need to be changed to match.

Reply:
sorry… my bad, I mistyped. As Ryan said, it should be “archive”!